front-app/.env.example

```diff
@@ -0,0 +1 @@
+PORT =
```

Comment:
Still missing the extra line

Reply:
![Screen Shot 2023-04-03 at 19 19 02](https://user-images.githubusercontent.com/92554753/229639563-75dedfab-3c07-4d34-b1f8-e0216b6f1d0a.png)